src/app/dataset/depublication/depublication.component.ts

```diff
@@ -134,6 +142,20 @@ export class DepublicationComponent extends DataPollingComponent {
         return deletionInfo.recordId !== recId;
       });
     }
+    this.checkAllAreSelected();
+  }
+
+  /** checkAllAreSelected
+  /*  sets allSelected variable according to check states
+  */
+  checkAllAreSelected(): void {
```

Comment:
I found the following interesting flow.
1. remove all records from the list.
2. select the 'select all' checkbox.
3. add records from a file.
4. All new records are deselected, but the select all checkbox is still selected.
It's not very dramatic, but still...